presentation-compiler/test/dotty/tools/pc/tests/inlayHints/InlayHintsSuite.scala

```diff
@@ -664,4 +664,105 @@ class InlayHintsSuite extends BaseInlayHintsSuite {
         |case class ErrorMessage(error)
         |""".stripMargin
     )
-}
\ No newline at end of file
+
+  @Test def `anonymous-given` =
+    check(
+     """|package example
+        |
+        |trait Ord[T]:
+        |  def compare(x: T, y: T): Int
+        |
+        |given intOrd: Ord[Int] with
+        |  def compare(x: Int, y: Int) =
+        |    if x < y then -1 else if x > y then +1 else 0
+        |
+        |given Ord[String] with
+        |  def compare(x: String, y: String) =
+        |    x.compare(y)
+        |
+        |""".stripMargin,
+     """|package example
+        |
+        |trait Ord[T]:
+        |  def compare(x: T, y: T): Int
+        |
+        |given intOrd: Ord[Int] with
+        |  def compare(x: Int, y: Int)/*: Int<<scala/Int#>>*/ =
+        |    if x < y then -1 else if x > y then +1 else 0
+        |
+        |given Ord[String] with
+        |  def compare(x: String, y: String)/*: Int<<scala/Int#>>*/ =
+        |    /*augmentString<<scala/Predef.augmentString().>>(*/x/*)*/.compare(y)
+        |
+        |""".stripMargin
+    )
+
+  // TODO: Add a separate option for hints for context bounds
```

Comment:
Is that from original PR? We should remove it here and have na issue for it then